systemtest/playlist.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../TestConfig/playlist.xsd">
-	
+	<include>systemtest.mk</include>
 	<test>
```

Comment:
As you have defined a set of make variables (e.g., EXPORT, PERL...), do you want to use them in the test command?

Reply:
There are three failures that are unrelated and excluded in the #708.